lib/thor/actions/file_manipulation.rb

```diff
@@ -80,14 +80,14 @@ def get(source, *args, &block)
       config = args.last.is_a?(Hash) ? args.pop : {}
       destination = args.first
 
-      if source =~ %r{^https?\://}
+      render = if source =~ %r{^https?\://}
         require "open-uri"
+        URI.send(:open, source) { |input| input.binmode.read }
```

Comment:
Why we need the send?

Reply:
It uses `send` method because this is a private method in Ruby 2.4 and lower. 

## Ruby 2.5

```console
% ruby -ropen-uri -ve 'URI.open("https://github.com/erikhuda/thor")'
ruby 2.5.6p201 (2019-08-28 revision 67796) [x86_64-darwin17]
```

## Ruby 2.4

```console
% ruby -ropen-uri -ve 'URI.open("https://github.com/erikhuda/thor")'
ruby 2.4.7p357 (2019-08-28 revision 67796) [x86_64-darwin17]
-e:1:in `<main>': private method `open' called for URI:Module (NoMethodError)
```